libgalois/include/katana/GraphTopology.h

```diff
@@ -875,6 +875,8 @@ class KATANA_EXPORT EdgeTypeAwareBiDirTopology
     return Base::in().edges(N, edge_type);
   }
 
+  auto in_edges(Node N) const noexcept { return Base::in().edges(N); }
```

Comment:
Add `edges(N)` too.

Both `edges(N)` and `in_edges(N)` is already exposed in the base class `BasicBiDirTopoWrapper` but from what I understand, because the derived class adds a new polymorphic overload, the base class function is overridden. I'm not sure if there is a cleaner way to do this.